docs/library/layout/inset.md

```diff
@@ -1,6 +1,16 @@
 ---
 components:
     - rx.radix.themes.Inset
+
+Inset: |
```

Comment:
This one, may be nice to have some additional text/content on the card so the inset is more obvious